src/container/srv_target.c

```diff
@@ -1204,6 +1210,12 @@ cont_child_destroy_one(void *vin)
 		if (cont->sc_dtx_resyncing)
 			ABT_cond_wait(cont->sc_dtx_resync_cond, cont->sc_mutex);
 		ABT_mutex_unlock(cont->sc_mutex);
+
+		/* Make sure checksum scrubbing has stopped */
+		ABT_mutex_lock(cont->sc_mutex);
+		if (cont->sc_scrubbing)
+			ABT_cond_wait(cont->sc_scrub_cond, cont->sc_mutex);
```

Comment:
If scrubbing ULT is sleeping while holding ds_cont_child, you need to wakeup it.

Reply:
Fixed